### PR TITLE
issue #7645: Update doc for IllegalToken

### PR DESCRIPTION
**Pull request by HuGanghui**
issue: #7645: Update doc for IllegalToken
WechatIMG24

Ouput of default example:
$ cat config.xml
```xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
    <module name="TreeWalker">
        <module name="IllegalToken"/>
    </module>
```
    
$ cat Test.java
```java
public class Test {

    public void myTest() {
        outer: // violation
        for (int i = 0; i < 5; i++) {
            if (i == 1) {
                break outer;
            }
        }
    }
}
```

$ RUN_LOCALE="-Duser.language=en -Duser.country=US"
$ java $RUN_LOCALE -jar /var/tmp/checkstyle-8.31-SNAPSHOT-all.jar -c config.xml Test.java
Starting audit...
[ERROR] /Users/hgh/Downloads/my_file/IDEA_workspace/Leetcode_maven/engineering_ability/src/main/java/Test.java:4:14: Using 'outer:' is not allowed. [IllegalToken]
Audit done.
Checkstyle ends with 1 errors.

Ouput of configure the check to find token LITERAL_NATIVE example:
$ cat config.xml 
```xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
    <module name="TreeWalker">
        <module name="IllegalToken">
            <property name="tokens" value="LITERAL_NATIVE"/>
        </module>
    </module>
</module>
```
$ cat Test.java
```java
public class Test {

    public native void myTest();      // violation

}
```
$ RUN_LOCALE="-Duser.language=en -Duser.country=US"
$ java $RUN_LOCALE -jar /var/tmp/checkstyle-8.31-SNAPSHOT-all.jar -c config.xml Test.java
Starting audit...
[ERROR] /Users/hgh/Downloads/my_file/IDEA_workspace/Leetcode_maven/engineering_ability/src/main/java/Test2.java:3:12: Using 'native' is not allowed. [IllegalToken]
Audit done.
Checkstyle ends with 1 errors.

### CODE REVIEW

**Comment by HuGanghui** [comment]
sorry that config.xml does't show, as follows:
Ouput of default example:
$ cat config.xml
```xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
    <module name="TreeWalker">
        <module name="IllegalToken"/>
    </module>
```

Ouput of configure the check to find token LITERAL_NATIVE example:
$ cat config.xml
```xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
    <module name="TreeWalker">
        <module name="IllegalToken">
            <property name="tokens" value="LITERAL_NATIVE"/>
        </module>
    </module>
</module>
```

**Comment by HuGanghui** [comment]
sorry, i will fix this issue.
Proper commit message should adhere to the following rules:
834    1) Must match one of the following patterns:
835        ^Issue #\d+: .*$
836        ^Pull #\d+: .*$
837        ^(minor|config|infra|doc|spelling|dependency): .*$
838    2) It contains only one line of text
839    3) Must not end with a period, space, or tab
840    4) Commit message should be less than or equal to 200 characters
841
842The rule broken was: 1
843[INFO] 
844[ERROR] Tests run: 3189, Failures: 1, Errors: 0, Skipped: 2
845

**Comment by HuGanghui** [comment]
@strkkk  OK，I get it, I will fix these.

**Comment by HuGanghui** [comment]
@strkkk  ok, I will do it.

**Comment by HuGanghui** [comment]
> * Please squash comments into one (please do not forget to keep proper commit message)
> * Update first post with examples from code
@strkkk  I finish these things above~

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenCheck.java [hunk 1]
```diff
@@ -51,6 +51,17 @@
...
+ *     outer: // violation
+ *     for (int i = 0; i &lt; 5; i++) {
+ *         if (i == 1) {
+ *             break outer;
+ *         }
+ *     }
+ * }
+ * </pre>
  * <p>
  * To configure the check to find token LITERAL_NATIVE:
```
Comment (strkkk) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenCheck.java]: `to find token LITERAL_NATIVE:`  -> to report violation on token LITERAL_NATIVE:

**Comment by strkkk** [comment]
@HuGanghui sorry for confusion, there is one last item to correct.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenCheck.java (hunk 1 above)
Comment (HuGanghui) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenCheck.java]: ok, I will change it.

**Comment by HuGanghui** [comment]
> to find token LITERAL_NATIVE: -> to report violation on token LITERAL_NATIVE:

@strkkk  I change it and update the first post~

**Pull request merged**